packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -236,9 +236,9 @@ class QueryBuilder {
   setOrderIsUnique() {
     this.data.orderIsUnique = true;
   }
-  orderBy(exprGen: SQLGen, ascending: boolean = true) {
+  orderBy(exprGen: SQLGen, ascending: boolean = true, nullsFirst: boolean) {
```

Comment:
Technically this should also be `nullsFirst?: boolean` because it doesn't have to be specified. Huh, TypeScript and Flow use the same syntax for this 👍

Reply:
Actually it should be `nullsFirst: boolean | null = null` I think.

Reply:
You know what... Lets have it required. It should still be `| null` technically...